CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.1.0/)
 ## [Unreleased]
 ### Added
 - Github workflow for changelog verification ([#440](https://github.com/opensearch-project/flow-framework/pull/440))
+- Retry flaky tests ([#449](https://github.com/opensearch-project/flow-framework/pull/449))
```

Comment:
I don't think we need to add CI tweaks to our changelog. This is for user-facing changes in the plugin.  Also we don't have to really start adding anything until we cut the 2.12 branch at code freeze.